packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -265,6 +268,8 @@ describe('Signin', () => {
                     verified: false,
                     verificationReason: 'signup',
                     verificationMethod: 'email-otp',
+                    keyFetchToken: MOCK_KEY_FETCH_TOKEN,
+                    unwrapBKey: MOCK_UNWRAP_BKEY,
```

Comment:
I was a little surprised that this was need in this PR... almost feels like something in this PR actually fixed something... cause these values are used eventually.